### PR TITLE
Use comma delimited string for human stream share output

### PR DESCRIPTION
**Pull request by brianstrauch**
Checklist
---------
1. [CRUCIAL] Is the change for features that are already live in prod?
   * yes: ok

What
----
Follow up to https://github.com/confluentinc/cli/pull/2351, this time with the "Resources" and "Network Zonal Subdomains" fields of the `confluent stream-share` commands.

Test & Review
-------------
Updated integration tests

### CODE REVIEW

File: internal/stream-share/command_consumer_redeem.go [hunk 1]
```diff
@@ -13,21 +13,38 @@ import (
...
+	NetworkPrivateLinkData     string `human:"Network Private Link Data"`
+}
+
+type redeemSerializedOut struct {
+	Id                         string   `serialized:"id"`
+	ApiKey                     string   `serialized:"api_key"`
+	ApiSecret                  string   `serialized:"api_secret"`
+	KafkaBootstrapUrl          string   `serialized:"kafka_bootstrap_url"`
+	SchemaRegistryApiKey       string   `serialized:"schema_registry_api_key"`
+	SchemaRegistrySecret       string   `serialized:"schema_registry_secret"`
```
Comment (MuweiHe) [inline internal/stream-share/command_consumer_redeem.go:39]: api secret?

Comment (brianstrauch) [reply internal/stream-share/command_consumer_redeem.go:39]: Thoughts on renaming this field to "API secret"? @confluentinc/data-exchange

**Review by sgagniere — APPROVED** [review]
Would we be able to roll this into the table printing code in the future?

edit: I mean processing a string array into a comma delimited list for human output instead of creating two output structs

**Comment by brianstrauch** [comment]
> Would we be able to roll this into the table printing code in the future?
> 
> edit: I mean processing a string array into a comma delimited list for human output instead of creating two output structs

As long as we're guaranteed not to have exceptions... there might be a field in the future that we want to delimit with newlines instead of commas

**Pull request merged**